ivy.xml

```diff
@@ -19,5 +19,8 @@
     <dependency org="ome" name="formats-gpl" rev="${versions.bioformats}"/>
     <!-- Extensions for bundling into the release -->
     <dependency org="ome" name="OMEZarrReader" rev="${versions.OMEZarrReader}"/>
+    <!-- build-dev dependencies -->
+    <dependency org="joda-time" name="joda-time" rev="${versions.joda-time}"/>
```

Comment:
Is udunits `build-dev` specific?

Reply:
No.
I am upgrading version to see if it sorts out issue